<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Save_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>400</y>
      <width>85</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit_button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>400</y>
      <width>85</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="calorie_intake">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>202</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="date_input">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>242</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="weight_input">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>162</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="date_label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>247</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="calorie_label">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>202</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Calories</string>
    </property>
   </widget>
   <widget class="QLabel" name="weight_label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Weight</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>124</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_input">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
